tutorials/headless_rendering.md

```diff
@@ -0,0 +1,68 @@
+\page headless_rendering Headless Rendering 
+  
+It is often desirable to run simulation on a remote computer, such as
+a computer managed by cloud provider, in order to paralellize work or access
+specific compute resources. Simulated sensors that require GPU access has
+historically been difficult to use on a remote computer due to opengl's
+X server requirement. This issue can be resolved through installation and
+proper configuration of X, but the steps can be complex and error prone.
+
+An easier solution is through the use of [EGL](https://www.khronos.org/egl), which allows for the the creation of rendering surfaces without an X server. Ignition Gazebo has added support for EGL via the `--headless-rendering` command line option. For example:
```

Comment:
We should mention that this only works with Ogre2

Reply:
I mentioned OGRE2 in a couple places, see [53597c7](https://github.com/ignitionrobotics/ign-gazebo/pull/1386/commits/53597c79590e39cc32392827edfad9e47814570e)